### PR TITLE
CommonJS-fix: Store registration template payload as JS object on contextVars

### PR DESCRIPTION
**Pull request by samanehsan**
<h3>Purpose</h3>
Fixes issue where data was not unserializing in the template for OSF Standard Pre-Data Collection Registration.

<h3>Changes</h3>
Make `window.contextVars.regPayload` a JSON object instead of leaving it as a string so that `MetaData.ViewModel.unserialize` does not throw an error.

### CODE REVIEW

File: website/templates/metadata/register_1.mako [hunk 1]
```diff
@@ -47,6 +47,6 @@
     window.contextVars.node.children = ${[str(each) for each in children_ids]};
     window.contextVars.regTemplate = ${json.dumps(template_name or '')};
     window.contextVars.regSchema = ${schema};
-    window.contextVars.regPayload = ${json.dumps(payload)};
+    window.contextVars.regPayload = $.parseJSON(${json.dumps(payload)});
```
Comment (sloria) [inline website/templates/metadata/register_1.mako]: This shouldn't be necessary. Could you instead remove the JSONification of the already JSONified `payload` here: https://github.com/CenterForOpenScience/osf.io/blob/145ed8b009f0a1f36cd0fdab4c6850b1279af986/website/project/views/register.py#L69 ?

Comment (samanehsan) [inline website/templates/metadata/register_1.mako]: Sure, just made that change @sloria! :snowflake:

File: website/project/views/register.py [hunk 2]
```diff
@@ -66,7 +66,7 @@ def node_register_template_page(auth, **kwargs):
         payload = node.registered_meta.get(to_mongo(template_name))
         payload = json.loads(payload)
         payload = unprocess_payload(payload)
-        payload = json.dumps(payload)
+        payload = payload
```
Comment (sloria) [inline website/project/views/register.py]: This assignment is unnecessary.

**Comment by samanehsan** [comment]
Just fixed it @sloria :snowman:

**Pull request merged**